components/BottomSheet/src/MDCBottomSheetPresentationController.h

```diff
@@ -64,6 +64,15 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ This is used to set a custom height on the sheet view.
+
+ @note This will override setting the preferredContentSize if a positive value is passed.
+ @note If a non-positive value is passed then the sheet will open up to either half the screen
+ height or the size of the contentViewController whatever value is smaller.
+ */
```

Comment:
Looking at the implementation of MDCSheetContainerView.m, it looks like `truncatedPreferredSheetHeight` always considers the contentViewController's height when determining the maximum height. This implies that `preferredSheetHeight` will also be capped to the content's height, which the documentation doesn't yet indicate. We may want to add a test and documentation indicating this.

E.g. pseudo-code:

```
preferredSheetHeight = 1000
contentViewController.scrollView.contentSize.height = 500

assertEqual(sheet.height, 500) // Docs presently imply 1000
```